File: Learn the Basic/Pattern_printing/Star_Pyramid.cpp
#include<iostream>
using namespace std;

void Star_Pyramid(int n){
    for (int i=0;i<n;i++){
        for (int j=0;j<n-i-1;j++){
            cout<<" ";
        }
        for(int j=0;j<2*i+1;j++){
            cout<<"*";
        }
        for (int j=0;j<n-i-1;j++){
            cout<<" ";
        }
        cout<<endl;
    }
}
int main(){
    int n;
    cout<<"Enter the number of n: ";
    cin>>n;
    Star_Pyramid(n);
    return 0;
}

File: Learn the Basic/Pattern_printing/Star_Pyramid.py
# create a function
def Star_Pyramid(n):
    for i in range(n):
        for j in range(n-i-1):
            print(" ",end="")
        for j in range(2*i+1):
            print("*",end="")
        for j in range(n-i-1):
            print(" ",end="")
        print()
#call the function]
n=int(input("Enter the number of n: "))
Star_Pyramid(n)